charts/k8s-monitoring/docs/RBAC.md

```diff
@@ -279,7 +287,10 @@ Several components have specialized RBAC rules to perform their work. This docum
     - list
     - watch
 ```
-When deploying to an OpenShift cluster, these extra rules are added to enable access to special a SecurityContextConstraint:
+
+When deploying to an OpenShift cluster, these extra rules are added to enable access to special a
```

Comment:
This is passive voice. Does this mean the user should add these to enable access to a special constraint? I'm assuming it isn't automatic.

Reply:
Actually, it is automatic. Thanks for pointing that out. I'll fix.